content/features/03-graph-side-panel.adoc

```diff
@@ -3,10 +3,11 @@ title: "Graph: Side Panel"
 date: 2018-06-20T19:04:38+02:00
 draft: false
 image: features/graph-side-panel.png
-subTitle: "Get an overview with the Kiali side panel"
+subTitle: "Get an overview in the side panel"
```

Comment:
it sounds like it's repeating the text from another file here above?

Reply:
From what other file is this repeating from?

Reply:
probably a scrolling issue - I thought I saw this in 2 different files, but can't find that anymore.